Core/Config.cpp

```diff
@@ -969,7 +969,7 @@ static ConfigSetting controlSettings[] = {
 static ConfigSetting networkSettings[] = {
 	ConfigSetting("EnableWlan", &g_Config.bEnableWlan, false, true, true),
 	ConfigSetting("EnableAdhocServer", &g_Config.bEnableAdhocServer, false, true, true),
-	ConfigSetting("proAdhocServer", &g_Config.proAdhocServer, "myneighborsushicat.com", true, true),
```

Comment:
Seems myneighborsushicat.com is way more popular, why change the default?

Reply:
Well if they're doing custom adhoc server, their server won't be compatible with ProOnline-client and JPCSP or other PSP emulators right?

Reply:
No its possible to play versus PSP and Vita on them and I guess JPCSP too.

Reply:
I think silently changing the default may confuse people. Let's not do it in this PR, because it's unrelated to GetIPList - BUT! We could and should later indeed add a list of the most popular servers to choose from, along with descriptions of each. There we would include both myneighborsushicat and socom.cc. 

Possibly could even fetch the list from a json on ppsspp.org or something....

Reply:
We currently have public adhoc server list (only contains myneighborsushicat and socom.cc as of now) appended to the list output from GetIPList, but it's hardcoded.
It would be nice if it was taken from a json :)

Okay, I'll remove the default adhoc server changes

Reply:
> Well if they're doing custom adhoc server, their server won't be compatible with ProOnline-client/plugin and JPCSP or other PSP emulators right?
> 
> May be the owner, @Linblow can tell us what are the difference between socom.cc and other public adhoc server like sushicat?
> One thing i know is that there are players that are still confused where they can find people to play with, and socom.cc have a discord server linked from their website to do this.

So far, the only changes I made are the following:
- Identify user with his (IP, source port, MAC) trio instead of just (IP, MAC). 
Thus allowing multiple connections per ip **when** the client will be able to handle it (so it's useless for now).
- Added more games to crosslink databases (beware some games must be patched manually to allow crossplay).
- Refactored the code a little bit

Of course, it remains compatible with the current Prometheus client.

The web status frontend is the original Prometheus with a few HTML/CSS tweaks.
The `status.xml` page is valid XML that anyone can parse.

We also made (credits to MrChip) a Discord bot that provides a bunch of commands: `!status` command to see who's online, a command to generate a PPSSPP package (win/android) that is configured for the user who requested it, etc.

In the meantime, I decided to remake and restructure the whole adhoc server/client to bring critical features (and better packets, as you mentionned). Some of those features are: multiple connections per IP (max is set to 5 for now), UDP hole punching to forget network connection issues, global adhoc anticheat with game-specific cheat detectives (server-sided), user control access (ie. ban if detected cheating), game-specific patches for crossplay, etc.

The reason I don't incorporate all those changes directly into the current PPSSPP adhoc implementation is because the code is, in my opinion, too much packed and lacks flexibility. The new adhoc will be designed for two targets: PSP (CFW) and PPSSPP. Of course, target specific tweaks will be handled (eg. eat X cycles for PPSSPP).

For now, I'm making the base framework on which the new adhoc server/client is based. 
This framework must be cross-platform (PSP, Windows, Linux, etc).
It mainly provides a wrapper for the host sockets API (including PSP).

Note the new client implementation is based on the original Prometheus, and the official PSPNET librairies which I'm also reversing. As you can imagine, it's a lot of work. @ANR2ME Any contribution will be welcome in the future, especially since you've already fixed stuff in the current client.

Reply:
Btw how do you differentiatie whether the client is using the old packets or new one? since currently there is now way to findout which version is the server running. 
May be you implemented additional packet Opcode to retrieve server version after Login Opcode? (ie. if a client never use this Opcode it will be considered as old client)

Reply:
Exactly I'll implement new opcodes for the new adhoc client. That way, I know when a legacy Prometheus user is connecting. Originally, I didn't intend on keeping compatibility. Obviously, legacy Prometheus users will not benefit from most new features.
There will be a packet to get the server/client version.